src/frontend/Sider/Behandling/Inngangsvilkår/Målgruppe/MålgruppeVilkår.tsx

```diff
@@ -1,43 +1,53 @@
 import React from 'react';
 
-import { EndreMålgruppeForm } from './EndreMålgruppeRad';
+import styled from 'styled-components';
+
 import { målgruppeErNedsattArbeidsevne, målgrupperHvorMedlemskapMåVurderes } from './utils';
 import JaNeiVurdering from '../../Vilkårvurdering/JaNeiVurdering';
-import { DelvilkårMålgruppe } from '../typer/målgruppe';
+import { DelvilkårMålgruppe, MålgruppeType } from '../typer/målgruppe';
 import { Vurdering } from '../typer/vilkårperiode';
 
+const Container = styled.div`
+    display: flex;
+    gap: 2rem;
+`;
+
 // TODO: Rename til MålgruppeDelvilkår
 const MålgruppeVilkår: React.FC<{
-    målgruppeForm: EndreMålgruppeForm;
+    type: MålgruppeType;
+    delvilkår: DelvilkårMålgruppe;
     oppdaterDelvilkår: (key: keyof DelvilkårMålgruppe, vurdering: Vurdering) => void;
     feilmelding?: string;
-}> = ({ målgruppeForm, oppdaterDelvilkår }) => {
-    if (målgruppeForm.type === '') {
-        return null;
+}> = ({ type, delvilkår, oppdaterDelvilkår }) => {
+    const skalVurdereMedlemskap = målgrupperHvorMedlemskapMåVurderes.includes(type);
+    const skalVurdereDekketAvAnnetRegelverk = målgruppeErNedsattArbeidsevne(type);
+
+    if (skalVurdereMedlemskap || skalVurdereDekketAvAnnetRegelverk) {
```

Comment:
Tenker det er fint å returnere null først, då trenger man ikke ekstra tab for "return <Container.."

Dvs
```typescript
if (!(skalVurdereMedlemskap || skalVurdereDekketAvAnnetRegelverk)) {
  return;
}

return <Container>...
```